staging/src/k8s.io/apiserver/pkg/server/egressselector/config_test.go

```diff
@@ -133,86 +133,6 @@ egressSelections:
 			},
 			expectedError: nil,
 		},
-		{
-			name:       "v1beta1 using deprecated 'master' type",
```

Comment:
keep this test... it should continue working until the v1beta1 config is no longer supported

Reply:
Updated.